src/test/functional/shared/apiHelpers.ts

```diff
@@ -153,3 +153,14 @@ export const deleteUser = async (email) => {
     throw new Error(`Failed to delete user with email ${email}, http-status: ${e.response?.status}`);
   }
 };
+
+export const deleteAllTestData = async (testDataPrefix = '', userNames = [], roleNames = [], serviceNames = [], async = false) => {
+  try {
+    await axios.delete(
+      `${config.get('services.idam.url.api')}/testing-support/test-data?async=${async}&userNames=${userNames.join(',')}&roleNames=${roleNames.join(',')}&testDataPrefix=${testDataPrefix}&serviceNames=${serviceNames.join(',')}`
```

Comment:
Maybe it'd be better to build this request in a similar fashion to the other requests? using URLSearchParams object? :)

Reply:
Actually we can leave this for another day